src/main/kotlin/com/icat/orboarding/user/adapters/outbound/persistence/mysql/entities/RoleEntity.kt

```diff
@@ -0,0 +1,27 @@
+package com.icat.orboarding.user.adapters.outbound.persistence.mysql.entities
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import org.springframework.security.core.GrantedAuthority
+import javax.persistence.*
+
+@Entity
+@Table(name = "role")
+class RoleEntity(
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private val id: Int? = null,
```

Comment:
```suggestion
    @Id
    @Column(unique = true, nullable = false, length = 50)
    val id: String? = null,
```
Maninho, como todos as nossas outras entidades os identificadores são uuids, o que acha da gente deixar aqui tbm, para mantermos o padrão?

Reply:
Fechou manin, pode ser, alterado o/